client/actions/loginActions.js

```diff
@@ -67,9 +57,7 @@ export function resetPassword(body){
             .then((response) => {
                 dispatch(resetPasswordAction(response))
             })
-            .catch((err) => {
-                console.log("error========>>>>", err);
-            })
+            .catch({/** */})
```

Comment:
please don't swallow exceptions.
use this instead
catch((err) => { console.error(err);}